Documentation/contributing/release/backports.rst

```diff
@@ -179,6 +190,24 @@ Via GitHub web interface
    ``vRELEASE-backport-YYYY-MM-DD.txt`` file that was generated by the scripts
    above.
 
+   .. note::
+
+       The ``vRELEASE-backport-YYYY-MM-DD.txt`` file will include:
+
+          .. code-block:: RST
+
+                Once this PR is merged, you can update the PR labels via:
+                ```upstream-prs
+                $ for pr in AAA BBB ; do contrib/backporting/set-labels.py $pr done VVV; done
+                ```
+
+       The ``upstream-prs`` tag `is
+       <https://github.com/cilium/release/blob/3c5fc2bdc38f8d290349a612a03cc83655f57a51/pkg/github/labels.go#L26>`_
+       `required
+       <http://github.com/cilium/cilium/pull/12644#discussion_r460143295>`_, so
```

Comment:
We can probably remove this second link. It's unlikely anyone if going to notice that there are two links instead of one here anyway. (It also feels a bit weird to point the docs to the discussion rather than the discussion to the docs, but that's another concern :smile: )